lib/Bindings/Python/OMModule.cpp

```diff
@@ -349,6 +363,88 @@ Map::dunderGetItem(std::variant<intptr_t, std::string, MlirAttribute> key) {
   return dunderGetItemAttr(std::get<MlirAttribute>(key));
 }
 
+// Convert a generic MLIR Attribute to a PythonValue. This is basically a C++
+// fast path of the parts of attribute_to_var that we use in the OM dialect.
+static PythonPrimitive omPrimitiveToPythonValue(MlirAttribute attr) {
+  if (omAttrIsAIntegerAttr(attr)) {
+    auto strRef = omIntegerAttrToString(attr);
+    return py::int_(py::str(strRef.data, strRef.length));
```

Comment:
is there a reason, we need to go from int to str to `py::int_` ?
(Not super familiar with the pybind API)